### DISCUSSION TITLE
MOD 3.0: definition of profiles

### DISCUSSION THREAD

**Discussion by agbeltran** [sifrproject/MOD-Ontology#56] [General]
There has been previous work on defining what is the minimum metadata for a semantic artefact description.

We need to take the previous work forward and agree what MOD 3.0 will establish as minimum metadata.

Minimum metadata depends on the context, so it is likely that we will define a few profiles with minimum metadata for description, minimum metadata for making the description FAIR, etc.

This is a placeholder to keep the points of discussion.

**Comment by jonquet** [comment]
CC @dgarijo for this. Point discussed recently during All-Hands meeting.

**Reply by dgarijo** [reply]
Thanks! Once I have  final conclusions, I will share them here.

**Comment by biswanathdutta** [comment]
@agbeltran yes, we should look at the previous study conducted under FAIRImpact.
One more point: the minimum metadata should provide a core set of elements aligned with the core FAIR principles. Of course, on this minimum metadata topic, we brainstormed a lot in the FAIRImpact project.
Besides, the core set, we can have an extended set. The extended set will consist of all other non-core elements.

**Comment by jonquet** [comment]
Personnally, I would not necessarily "re-open" the pandora box of the minimal metadata. 
I would implement an MOD that can accept as much metadata properties as possible and that "incorportate" some profiles: 
- FAIRsFAIR minimum metadata profile
- DCAT/Dublin Core profile
- Schema.org profile
- DataCite profile
- Metashare profile
- etc. 
- and MOD "maximal metadata" profile

Search for this diagram in 
https://app.diagrams.net/#G1DN3YcXJzwS2tyiy8l_VOuIF1JmnCf50g#%7B%22pageId%22%3A%22zefHS8Miyx2UjzX9OqmE%22%7D
(tab: MOD2 proposition with selected properties)

<img width="504" alt="Capture d’écran 2023-10-10 à 15 48 11" src="https://github.com/FAIR-IMPACT/MOD/assets/7379183/550e1664-94e2-43ee-8a95-c5624b9cb275">

It shows all the possible properties MOD2 currently include (in red) plus the mapped ones (not in red). 
From this... we can make any profile...
